### PR TITLE
DOC: sparse doc fixups

### PR DESCRIPTION
**Pull request by TomAugspurger**

### CODE REVIEW

File: doc/source/whatsnew/v0.19.0.rst [hunk 1]
```diff
@@ -1236,6 +1236,7 @@ Operators now preserve dtypes
 - Sparse data structure now can preserve ``dtype`` after arithmetic ops (:issue:`13848`)
 
 .. ipython:: python
+   :okwarning:
```
Comment (jreback) [inline doc/source/whatsnew/v0.19.0.rst]: maybe better to make these code blocks?

Comment (TomAugspurger) [reply doc/source/whatsnew/v0.19.0.rst]: Probably. I'm still hoping someone will write a script to do this automatically someday. Just capture the output and freeze that.

Comment (jreback) [reply doc/source/whatsnew/v0.19.0.rst]: lol

**Comment by jreback** [comment]
thanks @TomAugspurger

File: doc/source/whatsnew/v0.19.0.rst (hunk 1 above)
Comment (jorisvandenbossche) [reply doc/source/whatsnew/v0.19.0.rst]: > I'm still hoping someone will write a script to do this automatically someday. Just capture the output and freeze that.

That's also my long term plan ;) 
It sounds as not too hard, but sphinx keeps being a mystery most of the time ..

**Pull request merged**